Gemfile

```diff
@@ -4,4 +4,7 @@ git_source(:github) { |repo| "git@github.com:#{repo}.git" }
 gemspec
 
 gem 'workarea-api', '>= 4.1.x'
-gem 'workarea', github: 'workarea-commerce/workarea'
+
+source 'https://gems.weblinc.com' do
```

Comment:
You'll need to add gem creds if you're going to reference the private gem server.

Reply:
i can't because i don't own the repo